db/seeds.rb

```diff
@@ -186,3 +186,5 @@
   r.location = boulder
   r.feed_format = "video"
 end.save!
+
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
Let's make sure that the seed script is **idempotent**. It should be able to be run more then once without breaking.